core/edit/edit_command_modifiers.talon-list

```diff
@@ -3,12 +3,18 @@ list: user.edit_modifier
 all: document
 paragraph: paragraph
 word: word
+word left: wordLeft
+word right: wordRight
 line: line
 line start: lineStart
 way left: lineStart
 line end: lineEnd
 way right: lineEnd
+up: lineUp
+down: lineDown
 file start: fileStart
 way up: fileStart
 file end: fileEnd
 way down: fileEnd
+left: left
```

Comment:
Everything works as expected, but adding left and right adds some more-or-less worthless commands to the grammar since it extends by a character. 

"cut left" "copy left" are pretty dubious.

I wonder if left / right / word left / up / down should be re-worked to be consistent with our arrows grammar e.g. copy 2 left, copy 2 up etc instead.

Reply:
Ensure this idea works with classic repeaters

Reply:
> Ensure this idea works with classic repeaters

@nriley FYI eg cut word left 3rd vs cut 3 word left will render different results for likely obvious reasons.

Since cut is destructive, only the final word would be on the clipboard with the classic repeaters, whereas cut 3 word left can be implemented to do what I would expect: three words on the clipboard. 

Copy/Selection actions work as expected with either.